src/main/java/teammates/common/util/Const.java

```diff
@@ -52,7 +52,7 @@
     public static final String NO_SPECIFIC_RECIPIENT = "No specific recipient";
 
     public static final String DISPLAYED_NAME_FOR_SELF_IN_COMMENTS = "You";
-    public static final String DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT = "Anonymous";
+    public static final String ANONYMOUS_USER = "Anonymous";
```

Comment:
I think we should name the variable as `DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT`. `ANONYMOUS_USER` can be misunderstood as an object, other than that Looks good to be reviewed by a senior dev.